### PR TITLE
create err , done msg when user submit to send feedback

### PR DESCRIPTION
**Pull request by moatazalaa14**

### CODE REVIEW

File: src/components/FeedbackForm/index.js [hunk 1]
```diff
@@ -51,12 +52,21 @@ export const FeedbackForm = () => {
             if (res.status === 201) {
               setInputVal({ value: '', errorMsg: '' });
               setDropDownValue({ ...dropDownValue, value: '' });
+              setDoneMsg({ value: 'Msg is Sended', status: true });
+              setInterval(() => {
+                setDoneMsg({ value: '', status: false });
+              }, 3000);
```
Comment (MAlngaawy) [inline src/components/FeedbackForm/index.js:58]: it doesn't handle like that!
there remove if condition and add `.then()` for success like you did `.catch()` for errors

File: src/components/FeedbackForm/index.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ export const FeedbackForm = () => {
     value: '',
     status: false
   });
+  const [doneMsg, setDoneMsg] = useState({ value: '', status: false });
```
Comment (MAlngaawy) [inline src/components/FeedbackForm/index.js:14]: `doneMessage` and `dropdownValue` and `inputValue` and `length` need to change  ---  consider these rules.
- the status is already _false_ if the state is empty or null and it becomes _true_ if it's not empty. so remove the status from (dropDownValue and doneMsg) state
- error messages should be in a separate state so, remove them from (length and inputValue) and make another state for every error message

**when you finish the above requerments you should have just one state as an object it's `length` state**

and finnaly handle the code below to work with it

File: src/components/FeedbackForm/index.js [hunk 3]
```diff
@@ -51,12 +52,21 @@ export const FeedbackForm = () => {
...
-            setInputVal({ value: '', errorMsg: '' });
-            setDropDownValue({ ...dropDownValue, value: '' });
+            console.log(err);
+            setDoneMsg({
+              value: 'you can send only 5 feedback in day',
+              status: false
+            });
+            setInterval(() => {
+              setDoneMsg({ value: '', status: false });
+            }, 3000);
```
Comment (MAlngaawy) [inline src/components/FeedbackForm/index.js:69]: why you used setInterval here!

**Review by MAlngaawy — CHANGES_REQUESTED** [review]
please, consider these changes Mu'taz

**Review by MAlngaawy — CHANGES_REQUESTED** [review]
and please add a description, why you open this pull request because I didn't get why it opened

File: src/components/FeedbackForm/index.js (hunk 3 above)
Comment (moatazalaa14) [reply src/components/FeedbackForm/index.js:69]: to make msg empty after 3 seconds and make div hidden

Comment (Nilegfx) [reply src/components/FeedbackForm/index.js:69]: 1. if this is a requirement, then it should be `setTimeout` not `setInterval`

2. this is not a requirement so don't implement it.

File: src/components/FeedbackForm/index.js (hunk 2 above)
Comment (moatazalaa14) [reply src/components/FeedbackForm/index.js:14]: For doneMessage :-

- status for displaying div or not but value for Msg whether it's done or no

Comment (MAlngaawy) [reply src/components/FeedbackForm/index.js:14]: The SOLID principles are a set of software design principles, the first principle is 

**S - Single Responsibility Principle**
- Any function must be responsible for doing only ONE thing. Only one potential change in the software’s specification should be able to affect the specification of the class.

implement it on the react.js state :arrow_up:

**Comment by Nilegfx** [comment]
I am closing this as it is not a prioritized feature.

**Pull request closed without merging**